docker/build-inaugurator.dockerfile

```diff
@@ -27,9 +27,6 @@ RUN yum install -y \
     yum -y clean all
 
 
-RUN wget http://linuxsoft.cern.ch/cern/centos/7/updates/x86_64/Packages/kernel-3.10.0-514.6.1.el7.x86_64.rpm
```

Comment:
@susyo not sure that u should remove it. What does exist in the container? I cannot see any kernel rpm installed there

Reply:
Does it have to take packages from the internet?